frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -35,7 +35,7 @@ const isDimensionDimensionMetric = (cols, strict = true) =>
     isMetric(cols[2])
 
 const isDimensionMetricMetric = (cols, strict = true) =>
```

Comment:
That said, this is pretty confusing, and I'm not 100% sure it's correct any more...